factories/receivers_aix.go

```diff
@@ -0,0 +1,166 @@
+// Copyright  observIQ, Inc.
```

Comment:
I strongly feel that the list of receivers should be limited to a subset that we will commit to supporting. Similar feeling for exporters.

Reply:
We need to define this, but I completely agree. I'm going to try and discuss with Ryan today.